english/hallway/hallway.md

```diff
@@ -0,0 +1 @@
+You begin walking down the hallway. The floor boards creak with every other step you take. You come across an open doorway, and inside it looks like a laboratory of some kind. There are many computers and strange machines humming away in the room.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.